movie_backend/movie_api/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+class Movie(models.Model):
```

Comment:
it would be nice to use comments to describe what this class does. Something like this:
```
 """
        states fields from class to be serialized
        attributes:
            category_name(string)- the name of the job category
            description(string)- the description of the category
            image(string)- the image to represent the category
        """
```